webpackjs/components/MicromobilityData/index.js

```diff
@@ -54,15 +56,14 @@ class MicromobilityData extends Component {
     // Queries the SODA API with the current date range query and stores data in state.
     // "dateRangeType" tells UI how to display date range to user ("current," "month," "range" or "all time")
     const resourceId = `7d8e-dm7r`;
-    const resourceId311 = `5h38-fd8d`;
     const tripSelectors = `avg(trip_duration)/60 as avg_duration_minutes, sum(trip_distance) * 0.000621371 as total_miles, avg(trip_distance) * 0.000621371 as avg_miles, count(trip_id) as total_trips`;
     const tripFilters =  `trip_distance * 0.000621371 >= 0.1 AND trip_distance * 0.000621371 < 500 AND trip_duration < 86400`;
 
     const dataByModeQuery = `SELECT vehicle_type, ${tripSelectors} WHERE start_time between ${dateQuery} AND ${tripFilters} GROUP BY vehicle_type`;
     const allModesQuery = `SELECT ${tripSelectors} WHERE start_time between ${dateQuery} AND ${tripFilters}`;
     const deviceCountScooterQuery = `SELECT DISTINCT device_id WHERE start_time between ${dateQuery} AND vehicle_type = 'scooter' LIMIT 1000000000`;
     const deviceCountBicycleQuery = `SELECT DISTINCT device_id WHERE start_time between ${dateQuery} AND vehicle_type = 'bicycle' LIMIT 1000000000`;
-    const threeOneOneQuery = `SELECT count(sr_type_code) as count WHERE sr_created_date between ${dateQuery} AND sr_type_code == "DOCKMOBI"`;
+    const threeOneOneQuery = `SELECT count(sr_type_desc) WHERE sr_created_date between ${dateQuery} AND (contains(upper(\`sr_type_desc\`), upper('dockless')))`;
```

Comment:
Unfortunately, the new socrata dataset does not have the `sr_type_code` field. Bummer indeed. I intend to discuss this with 311.

TIL that Socrata has a secret `contains()` query method that is not documented. I found it by inspecting the requests from the dataset UI.

Also, and this is quite weird, I had to remove the column name casting `as count` in order to make the query work. I tried a lot of different approaches and could not solve it. This dataset is either bugged out, on a different API version, or both.
<img width="710" alt="Screen Shot 2021-09-27 at 7 07 07 PM" src="https://user-images.githubusercontent.com/14793120/135002164-584d249c-ad78-46ce-84ec-cb3aba604bce.png">